packages/@yoda/flora/index.js

```diff
@@ -2,6 +2,31 @@
 
 /**
  * @module @yoda/flora
+ * @description The `flora` module provide protocol of cross process communication.
+ * exports 'Agent' constructor for generate `Agent` instance.
+ *
+ * ```js
+ * // create flora agent instance
+ * var flora = require('@yoda/flora');
+ * var agent = new flora.Agent('unix:/var/run/flora.sock');
```

Comment:
The example code should follow the standardjs coding style, so please remove the semi-colons :)